index.html

```diff
@@ -899,6 +899,10 @@ <h4>Event dispatch</h4>
                             <p>Otherwise (|event| is a <code>click</code> or <code>auxclick</code> event for which |userEvent| is a <code>pointerup</code> event that was dispatched uncaptured) let |target| be the nearest common inclusive ancestor of the corresponding <code>pointerdown</code> and <code>pointerup</code> targets in the DOM at the moment |event| is being dispatched.</p>
                         </li>
 
+                        <li>
+                          <p>If |event| is a <code>click</code> event, then <a href="https://html.spec.whatwg.org/#run-light-dismiss-activities">run light dismiss activities</a> given |event|.</p>
+                        </li>
```

Comment:
I don't quite understand how this works with customizable select's popup. That one is opened using mousedown, and then we'd a click, and that runs light dismiss. Maybe I'm just missing something